Explorer/Assets/Scripts/Global/Dynamic/ChatCommands/ShowEntityChatCommand.cs

```diff
@@ -0,0 +1,53 @@
+using Cysharp.Threading.Tasks;
+using DCL.Chat.Commands;
+using SceneRunner.Debugging;
+using SceneRunner.Debugging.Hub;
+using System.Threading;
+
+namespace Global.Dynamic.ChatCommands
+{
+    /// <summary>
+    /// Show entity components info.
+    ///
+    /// Usage:
+    ///     /show-entity *scene* *entity*
+    /// </summary>
+    public class ShowEntityChatCommand : IChatCommand
+    {
+        public string Command => "show-entity";
+        public string Description => "<b>/show-entity <i><scene> <entity></i></b>\n  Show entity components info";
```

Comment:
I would mark this one as internal. Its not useful for end users

Reply:
Will do, makes sense 👌🏻